### DISCUSSION TITLE
I'm sorry, but how to install this tool?

### DISCUSSION THREAD

**Discussion by tejas-2232** [DevToys-app/DevToys#1186] [General]
Hi all, I'm not able to find a way to install DevToys app.

I just read about this tool in one blog post posted on DEV platform. I'm eager to try it.

Thanks in advance for your help.

**Comment by veler** [comment]
Hi,
It depends, are you on Windows, Mac or Linux? Also, if you're willing to wait ~12h, our website will indicate it soon (https://devtoys.app)

**Reply by badcel** [reply]
@veler Regarding website: The download link for the portable Linux GUI binaries actually links to the CLI version.

I tried the released binary on fedora 40 and it works like a charm. Thank you!

**Reply by veler** [reply]
Indeed! Thanks for noticing it. I just updated the website to fix it.

**Comment by tejas-2232** [comment]
> Hi,
> It depends, are you on Windows, Mac or Linux? Also, if you're willing to wait ~12h, our website will indicate it soon (https://devtoys.app)

Hello, 
I use Linux and Windows. 
Okay sure. I'll wait. 
Thank you.

**Reply by veler** [reply]
Hi,
It's been more than 12h now. It's available here: https://devtoys.app/download
